SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -152,7 +155,43 @@ class CSourceBase_TurbSA : public CNumerics {
     Jacobian_i[0] = 0.0;
 
     /*--- Evaluate Omega with a rotational correction term. ---*/
-
+    // pgomega by Xiao He 2022
+      if (options.pgomga) {
+        
+        /*!\brief  PG-OMEGA*/
+        su2double dpds = 0, dpomg = 0, omg = 0, velocityMag = 0;
+        su2double Velocity_Rel[3] = {0.0, 0.0, 0.0};
+        su2double Vorticity_Rel[3] = {0.0, 0.0, 0.0};
+        su2double RotationalVelocity[3] = {0.0, 0.0, 0.0};
+        su2double RotationVelocityCrossR[3] = {0.0, 0.0, 0.0};
```

Comment:
```suggestion
        su2double RotationVelocityCrossR[nDim] = {0.0};
```

Reply:
or, since you are already doing static allocation: 
```suggestion
        su2double RotationVelocityCrossR[MAXNDIM];
```
and you do not rely on the initial value being 0 as far as I see